lib/client.js

```diff
@@ -587,7 +586,7 @@ class Client extends events.EventEmitter {
 
         await this.#connect();
 
-        const execOptions = DefaultExecutionOptions.create(options, this);
+        const _execOptions = DefaultExecutionOptions.create(options, this);
```

Comment:
Why do we rename this constant?

I see we renamed it, but I can see no changes in code that uses it. Actually, is any code using it? If not, maybe it's better to delete this instead of renaming?

Reply:
We will have to use this variable in near future, so I just left it here. Rename was due to eslint rules, which I updated to allow unused variables to start with ``_``.